src/main/webapp/views/COMMONS/searchBarVendors.html

```diff
@@ -0,0 +1,32 @@
+<div class="row justify-content-md-center">
+  <div class="col-12">
+    <div class="card border-primary">
+      <h5 class="card-header">Find the nearest vendor</h5>
+      <div class="card-body">
+        <div id="search-form">
+          <div class="row">
+            <div class="col">
+              <div class="form-group">
+                <label for="distance">Distance in meters:</label>
+                <input type="number" class="form-control" id="distance" aria-describedby="distance"
+                  placeholder="Distance" value="1000">
+              </div>
+              <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="" id="delivery">
+                <label class="form-check-label" for="delivery">
```

Comment:
Inputs are missing the name attr.

Reply:
I was planning to submit this form via JS, in that case we can access the element values via id to create the body of the request.

Reply:
Oh, I see, thanks for the clarification!